documentation/external/BookAnthology.json

```diff
@@ -0,0 +1,139 @@
+  "doi" : "http://example.org/doi/1231/98765",
+  "doiRequest" : {
+    "type" : "DoiRequest",
+    "status" : "REQUESTED",
+    "modifiedDate" : "2020-09-23T09:51:23.044996ZZ",
+    "createdDate" : "2020-09-23T09:51:23.044996ZZ",
+    "messages" : [ {
+      "type" : "DoiRequestMessage",
+      "text" : "Some Text",
+      "author" : "SomeAuthor",
+      "timestamp" : "2020-09-23T09:51:23.044996ZZ"
+    } ]
+  },
+  "files" : {
+    "type" : "FileSet",
+    "files" : [ {
+      "type" : "File",
+      "identifier" : "5032710d-a326-43d3-a8fb-57a451873c78",
+      "name" : "new document(1)",
+      "mimeType" : "application/pdf",
+      "size" : 2,
+      "license" : {
+        "type" : "License",
+        "identifier" : "NTNU-CC-BY-4.0",
+        "labels" : {
+          "no" : "CC-BY 4.0"
+        },
+        "link" : "http://example.org/license/123"
+      },
+      "administrativeAgreement" : true,
+      "publisherAuthority" : true,
+      "embargoDate" : "2020-09-23T09:51:23.044996ZZ"
+    } ]
+  },
+  "handle" : "http://example.org/handle/123",
+  "identifier" : "c443030e-9d56-43d8-afd1-8c89105af555",
+  "indexedDate" : "2020-09-23T09:51:23.044996ZZ",
+  "link" : "http://example.org/link",
+  "metadata" : {
+    "abstract" : "En lang streng som beskriver innholdet i dokumentet metadataene omtaler.",
+    "contributors" : [ {
+      "type" : "Contributor",
+      "identity" : {
+        "type" : "Identity",
+        "id" : "http://example.org/person/123",
+        "name" : "Navnesen, Navn",
+        "nameType" : "Personal",
+        "orcId" : "orc123",
+        "arpId" : "arp123"
+      },
+      "affiliations" : [ {
+        "type" : "Organization",
+        "id" : "http://example.org/org/123",
+        "labels" : {
+          "no" : "Eksempelforlaget"
+        }
+      } ],
+      "role" : "Creator",
+      "sequence" : 0,
+      "correspondingAuthor" : true,
+      "email" : "nn@example.org"
+    } ],
+    "description" : "En streng som beskriver innholdet i dokumentet på en annen måte enn abstrakt",
+    "language" : "http://example.org/norsk",
+    "mainTitle" : "Hovedtittelen",
+    "metadataSource" : "https://example.org/doi?doi=123/123",
+    "npiSubjectHeading" : "010",
+    "reference" : {
+      "date" : {
+        "type" : "PublicationDate",
+        "year" : "2020",
+        "month" : "4",
+        "day" : "7"
+      },
+      "isbn" : [ "9780201309515" ],
+      "originalDoi" : "https://123123/213123.com",
+      "pages" : {
+        "type" : "MonographPages",
+        "introduction" : {
+          "type" : "Range",
+          "begin" : "i",
+          "end" : "xx"
+        },
+        "pages" : "221",
+        "illustrated" : true
+      },
+      "peerReviewed" : true,
+      "publisher" : {
+        "type" : "OriginalPublishingChannelDto",
+        "name" : "My publisher dot com",
+        "level" : "LEVEL_0",
+        "openAccess" : false,
+        "peerReviewed" : true
+      },
+      "series" : {
```

Comment:
https://unit.atlassian.net/browse/NP-1862: 
> (...) Det kan være mismatch mhp nivå mellom utgiver og serie, men det er slik virkeligheten er. Seriens vit. nivå overkjører utgivers vit. nivå mht om publikasjonen kan rapporteres til NVI.

Jeg tror vi må ta vare på mer data for valgt serie også. Helst at vi har samme felter her som i `publisher`. Begge disse skal bare populeres med data fra oppslag mot kanalregister, så vi burde kanskje ta vare på all data derfra:
```
{
  "title":"Nature Genetics",
  "onlineIssn":"1061-4036",
  "printIssn":null,
  "level":2,
  "openAccess":false,
  "url":"http://www.nature.com/ng/"
}
```